app/controllers/admin/accounting/transactions_controller.rb

```diff
@@ -29,6 +29,18 @@ def create
     @loan = Loan.find(transaction_params[:project_id])
     authorize(@loan, :update?)
     @transaction = ::Accounting::Transaction.new(transaction_params)
+
+    # the txn model must be able to save managed txns before
```

Comment:
I didn't love adding this logic to a controller action. I tried to figure out a way at the model validation level to distinguish between an imported qb txn and a txn that had been created via the Madeline txn form. I didn't find a way that always works (e.g. we have to be able to create 'Managed' txns before the closed book date when running the full fetcher). I also looked at limiting in the browser, but our date-picker is customized text field, rather than a :date field that has min built in, so I didn't go that route. Happy to try something with the datepicker, or something else if it you think it would be worth it.